### PR TITLE
list structure

### PR DESCRIPTION
**Pull request by ibr5500**
# In this project I have:
- Set up webpack configurations.
- Set up linters configurations.
- Added dynamic list tasks using objects.
- Styled the layout to make it close to the original design. 
- Updated the READEME file.

### CODE REVIEW

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
### Hi STUDENT,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight!
- Good commits ✔️

# Required Changes ♻️ ⛔️
- [ ] Let's add a descriptive PR summary to show what changes you made.
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibr5500** [comment]
Hello, @EricMbouwe I have made the required changes.
Thank you for the code review.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @ibr5500,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
>- [x] As the previous reviewer suggested. Please kIndly add a descriptive PR title and description. 
:x: Wrong
Screenshot from 2022-04-28 12-45-28

Here is a demo of how to add a descriptive PR Title and Summary Below:

https://user-images.githubusercontent.com/52934128/165744267-9cc4c35b-143b-4c9a-863a-a88e47a8cec2.mp4


> 
> ## [Optional] suggestions:
>- [x] From now on,please kindly make use of imperative mood in the subject line for your commit messages.
:x: Wrong
Screenshot from 2022-04-28 12-42-03
> ### Good Commit Messages:
>- [x]  Add fix for dark mode toggle state
>- [x] Update README file.
>- [x] Add styles to h1.
>
>
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by ibr5500** [comment]
Thank you, @topeogunleye, for the code review.
I have made the required changes. 
I would like to thank you for the video, it was really helpful.

File: README.md [hunk 1]
```diff
@@ -1 +1,73 @@
...
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Please give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (rustam-tolipov) [inline README.md]: - [x] [OPTIONAL] You have linked the MIT license however there is no file. Please kindly add the MIT.md file to your project 👍

**Review by rustam-tolipov — APPROVED** [review]
## Hello @ibr5500 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by ibr5500** [comment]
Yes, I have forgotten to delete that.
Thank you, @Rustamxon7, for the code review.

**Pull request merged**